packages/next/lib/data.ts

```diff
@@ -26,7 +26,7 @@ export function createHook(
     throw new Error('key not provided to createHook options.')
   }
   return function useData(...args: Array<string | number>) {
-    const router: import('next-server/lib/router/router').default = useContext(
+    const router: import('next-server/lib/router/router').PublicRouterInstance = useContext(
```

Comment:
Just curious why this is written this way?

Reply:
To avoid importing the file at all, in this case we only need the type.

Reply:
iirc, TypeScript strips out empty import lines (e.g. those whose imports are all types)

Reply:
That's all I can tell you haha, I can't really appreciate another difference here

Reply:
:+1: just looked out of place considering all the other imports are at the top. Doesn't matter as far as I know.